instana/tracer.py

```diff
@@ -109,15 +121,15 @@ def start_span(self,
 
         return span
 
-    def inject(self, span_context, format, carrier):
+    def inject(self, span_context, format, carrier, disable_w3c_trace_context=True):
         if format in self._propagators:
-            return self._propagators[format].inject(span_context, carrier)
+            return self._propagators[format].inject(span_context, carrier, disable_w3c_trace_context)
 
         raise ot.UnsupportedFormatException()
 
-    def extract(self, format, carrier):
+    def extract(self, format, carrier, disable_w3c_trace_context=True):
```

Comment:
I reversed the default value here as well. I prefer to keep the flag, because I have better visibility on which instrumentation supports or does not support it yet

Reply:
Fair point 👍